controllers/mod/artefacts_metadata.rb

```diff
@@ -0,0 +1,87 @@
+class ArtefactsMetadataController < ApplicationController
+  namespace "/artefacts" do
+    # Get all Semantic Artefacts
+    get do
+      check_last_modified_collection(LinkedData::Models::SemanticArtefact)
+      attributes, page, pagesize = settings_params(LinkedData::Models::SemanticArtefact).first(3)
+      pagesize ||= 20
+      attributes = LinkedData::Models::SemanticArtefact.goo_attrs_to_load([]) if includes_param.first == :all
+      artefacts = LinkedData::Models::SemanticArtefact.all_artefacts(attributes, page, pagesize)
+      reply artefacts
```

Comment:
is this an hydra page ?

Reply:
yes it's hydra page, it's implemented in the response of the all_artefacts method: https://github.com/ontoportal-lirmm/ontologies_linked_data/blob/068c0b8c64160e5dcf807d470240b4fd9294c9c8/lib/ontologies_linked_data/models/mod/semantic_artefact.rb#L170